src/OpenTelemetry/Metrics/View/Aggregator.cs

```diff
@@ -0,0 +1,68 @@
+// <copyright file="Aggregator.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Metrics
+{
+    public enum Aggregator
```

Comment:
it'd be bit of a learning curve for an end user to pick the right aggregator per instrument.

Reply:
Victor - this mostly follows Views Spec. Its best to raise this in spec.